docs/detections/prebuilt-rules/tune-rule-signals.asciidoc

```diff
@@ -146,6 +147,17 @@ exception with the port number
 exception with the port number
 (`destination.ip is 172.16.0.0/12` and `destination.port is 445`).
 
+[float]
+[[tune-indicator-rules]]
+=== Tune indicator match rules
+
+Take the following steps to tune indicator match rules:
+
+* Specify a detailed query as part of the indicator index query. Every result returned from the indicator index query will be used by the detection engine to subsequently query the index patterns defined in your rule definition. Using no query or the wildcard `*.*` query will result in your rule executing potentially very large queries.
+* Limit your rule additional look-back time to as short a duration as possible, and no more than 24 hours.
```

Comment:
Not sure whether "rule" needs to be possessive (e.g., "Limit your rule's additional look-back time...).

Reply:
Good eye - thank you!